opsdroid/connector/slack/__init__.py

```diff
@@ -144,7 +144,9 @@ def __init__(self, config, opsdroid=None):
         _LOGGER.debug(_("Looking up sender username."))
         try:
             user_info = await self.lookup_username(message["user"])
-        except ValueError:
+        except (ValueError, KeyError) as err:
+            if err.__class__ == KeyError:
```

Comment:
I'm not sure this check is necessary. Either way the username lookup has failed.

Reply:
Right. Thanks